packages/glsp-sprotty/src/api-override.ts

```diff
@@ -198,3 +199,53 @@ export interface IActionDispatcher extends SIActionDispatcher {
 }
 
 export type IActionDispatcherProvider = () => Promise<IActionDispatcher>;
+
+/**
+ * Data structure that is passed to the `onCommandExecuted` event.
+ */
+export interface CommandExecutionData {
+    /** The command that has been executed successfully */
+    command: ICommand;
+    /** The new model root after the command execution */
+    newRoot: GModelRoot;
+}
+
+export interface ICommandStack extends SICommandStack {
+    /**
+     * Executes the given command on the current model and returns a
+     * Promise for the new result.
+     *
+     * Unless it is a special command, it is pushed to the undo stack
+     * such that it can be rolled back later and the redo stack is
+     * cleared.
+     */
+    execute(command: ICommand): Promise<GModelRoot>;
```

Comment:
Any reason you re-specified this method? Seems like we could just take it from the super interface.

Reply:
I re-specified every method that uses the sprotty SModel type (e.g. SModelRootImpl) instead of our gmodel types (GModelRoot). 

If we override the interface anyway we can also correct the types, witch improves IDE support. (We already do that for other classes/interfaces as well like the mouse listener or the vnode decorator.